README.md

```diff
@@ -35,193 +37,199 @@ unit that communicates with the cluster by relating to the
 
     juju-quickstart apache-hadoop-spark
```

Comment:
Let's change this to either:

``juju deploy apache-hadoop-spark``

or

``juju deploy bundle/apache-hadoop-spark``